### ISSUE TITLE
Update importer parse_date() method for new standard

### ISSUE THREAD

**Issue by jeancochrane** [security-force-monitor/sfm-cms#703]
SFM is working on a new data standard for storing dates in the spreadsheet. When this standard is settled, update the `parse_date()` method in the importer to accommodate it:

https://github.com/security-force-monitor/sfm-cms/blob/ce84206da1f43fc5a68ce483993165484a09f470/sfm_pc/management/commands/import_google_doc.py#L369-L396

Also update portions of the code that use the date part fields, which will be deprecated as well:

https://github.com/security-force-monitor/sfm-cms/blob/ce84206da1f43fc5a68ce483993165484a09f470/sfm_pc/management/commands/import_google_doc.py#L849-L867

https://github.com/security-force-monitor/sfm-cms/blob/ce84206da1f43fc5a68ce483993165484a09f470/sfm_pc/management/commands/import_google_doc.py#L1855-L1873

**Comment by hancush** [comment]
Per Tom:

> We're not able to do that fix in this round. Largely, it would take more cleaning/checking than we have the time to do this month.

**Comment by tlongers** [comment]
@hancush did we ultimately do this as part of resolving other issues?

**Comment by hancush** [comment]
@tlongers No – Last I heard, you all didn't feel date formatting was ready for primetime on your end, so we punted on this. Is that still the case?

**Comment by tlongers** [comment]
Correct, and thanks for clarifying. We weren't convinced by the robustness of  our own proposed approach of adding additional trailing hypens to partial dates ways that avoid apreadsheet auto-formatting e.g.  `2020--`, `2020-10--`. Still something to ponder.
